Bricks/strings/printf.h

```diff
@@ -33,6 +33,9 @@ SOFTWARE.
 namespace current {
 namespace strings {
 
+#ifndef CURRENT_WINDOWS
```

Comment:
I suppose it should be supported by any version of GCC/Clang that knows about c++11, so adding some specific version guards may be an overkill.